.template_system_tests.yml

```diff
@@ -0,0 +1,24 @@
+parameters:
+  module_name: ""
+  os_name: ""
```

Comment:
`os_name` is not a precise name, I think, because the options are linux, win32, win64 (right?)
But in reality both win32 and win64 are executed on the same OS (Windows 10 64-bit).

A more precise term would be `platform`.